project/src/const.js

```diff
@@ -0,0 +1,6 @@
+export const AppRoute = {
+  MAIN: '/',
+  SIGN_IN: '/login',
+  FAVORITES: '/favorites',
+  ROOM: '/offer/:id',
+};
```

Comment:
```suggestion
const AppRoute = {
  MAIN: '/',
  SIGN_IN: '/login',
  FAVORITES: '/favorites',
  ROOM_$ID: '/offer/:id',
};

export { AppRoute };
```
Экспорты только внизу файла

`$` в названии для роутеров конвеншин для обозначения динамических роутеров